### ISSUE TITLE
[Bug?] SimpleMDE creates a link with certain text

### ISSUE THREAD

**Issue by nivv** [sparksuite/simplemde-markdown-editor#111]
Just notcied when I started to actually write something :smile: This seems to only be visual. There's no links when saving

Check out this weird gif.
![simplemde](https://cloud.githubusercontent.com/assets/4033930/9986112/6e4f6102-6038-11e5-8b74-dcc0a2f092ec.gif)

**Comment by ADmad** [comment]
I am having the same problem. Random words are being styled as links (only visual no actual links are created). Seems to have started occurring since the latest release.

**Comment by brondsem** [comment]
This is due to a bug in the URL regex in CodeMirror's gfm mode that is currently being used by SimpleMDE.  (It was a bug in my pull-request to tweak that URL regex)  But it was fixed when it got merged into CodeMirror's master branch, so @WesCossick if you just update that file to the latest version it should take care of it.

**Comment by WesCossick** [comment]
It's already been fixed in the development branch of SimpleMDE. The fix will go out with v1.7.2.

**Comment by ADmad** [comment]
@WesCossick Is there an ETA for the next release? This visual anomaly is confusing to end users and given that it affects everyone would be great if you could make a new release soon.

**Comment by WesCossick** [comment]
No ETA, but the milestone for 1.7.2 is everything that needs to be completed before the next release. If you or others would like to contribute PRs for these changes, it would expedite the release.

**Comment by ADmad** [comment]
I wish I could help but I am a PHP guy and my JS skills are limited.

Guess I will just have to downgrade or use the development branch and wait until 1.7.2 is released.

**Comment by nivv** [comment]
@WesCossick Is there a reason why you can't release hot fixes like this when they are "ready"? I'm just curious, I'm new to this type of workflow.

**Comment by ADmad** [comment]
@nivv Very good question :)

**Comment by WesCossick** [comment]
Publishing each new release requires a variety of steps and extra testing work. I have multiple full time jobs/responsibilities and I would prefer to bundle changes together into larger releases to reduce my total amount of work.

For now, you can download the master branch and run gulp. It will download the CodeMirror dependencies automatically, including the updated regex.

**Comment by nivv** [comment]
@WesCossick I see! That makes a lot of sense. Thanks for the info :smile_cat:

**Comment by WesCossick** [comment]
v1.7.2 has been released and published to [NPM](http://npmjs.com/package/simplemde).

**Comment by ADmad** [comment]
Thanks @WesCossick.
